core/src/test/java/com/linecorp/armeria/server/ServiceRequestContextTest.java

```diff
@@ -87,6 +97,12 @@ void mapCurrent() {
             assertThat(onEnterExitStack).hasSize(1);
             assertThat(ServiceRequestContext.mapCurrent(c -> "foo", () -> "bar")).isEqualTo("foo");
```

Comment:
I think these tests are too loose, and especially with the double-stack, the new test code is not testing anything interesting - we only assert that any context is mounted and mapped, but it may not necessarily be the right context. `c == ctx ? "foo" : "bar"`? Or add/read attributes? Not sure which is less tedious.

Reply:
That's a good point. `c == ctx ? "foo" : "bar"` is fine for me. Thanks!